scout/constants/variant_tags.py

```diff
@@ -9,7 +9,20 @@
     'unknown'
 )
 
-CONSERVATION = ('NotConserved', 'Conserved')
+CONSERVATION = {
+    'dbNSFP_GERP___RS' : {
+        'conserved_min' : 2,
+        'conserved_max' : 10
+    },
+    'dbNSFP_phastCons100way_vertebrate': {
+        'conserved_min' : 0.8,
+        'conserved_max' : 100
```

Comment:
I'll assume you have checked these values, or that they are the same as the previous terms: as far as I remembered, phastCons used to give a P-val? Is it scaled or the -log P perhaps to be more similar to the others?

Reply:
I took these values from here: https://github.com/Clinical-Genomics/MIP/blob/master/templates/rank_model_cmms_-v1.20-.ini

as also @henrikstranneheim  suggested. Do they make any sense to you?